### PR TITLE
Cybernetic Organs

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Ports over a number of cybernetic organs from TG: https://github.com/tgstation/tgstation/pull/30382 https://github.com/tgstation/tgstation/pull/30382

Cybernetic organs include: heart, lungs, liver, and kidneys.

img

Regular cybernetic organs are the same, performance level, as regular organs, with the advantage of being immune to germ accumulation, that said, they're vulnerable to EMP, as all robotic organs are.

Additionally, there is an upgraded set of cybernetic lungs that are slightly better at handling CO2 and plasma in the air, in addition to being more resistant to cold gasses breathed in.

Also went ahead and enabled augmentation of the head.

code wise: moved a number of itnernal organs to their own unique files

:cl: Fox McCloud
tweak: Enabled augmentation of the head
add: Adds cybernetic heart, lungs, liver, and kidneys to R&D and Robotics; also adds cybernetic upgraded lungs
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
Awesome. Augs FTW!

File: code/modules/surgery/organs/heart.dm [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+				H.adjustFireLoss(-cursed_heart.heal_burn)
+				H.adjustOxyLoss(-cursed_heart.heal_oxy)
+
+/obj/item/organ/internal/heart/cybernetic
+	name = "cybernetic heart"
+	desc = "An electronic device designed to mimic the functions of an organic human heart. Offers no benefit over an organic heart other than being easy to make."
+	icon_state = "heart-c-on"
+	icon_base = "heart-c"
+	dead_icon = "heart-c-off"
+	origin_tech = "biotech=5"
```
Comment (Crazylemon64) [inline code/modules/surgery/organs/heart.dm]: should this be `4` or `5`?

Comment (Fox-McCloud) [reply code/modules/surgery/organs/heart.dm]: definitely 5. See regular heart's biotech value.

File: code/modules/surgery/organs/kidneys.dm [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	if(coffee)
+		if(is_bruised())
+			owner.adjustToxLoss(0.1 * PROCESS_ACCURACY)
+		else if(is_broken())
+			owner.adjustToxLoss(0.3 * PROCESS_ACCURACY)
+
+/obj/item/organ/internal/kidneys/cybernetic
+	name = "cybernetic kidneys"
+	icon_state = "kidneys-c"
+	desc = "An electronic device designed to mimic the functions of human kidneys. It has no benefits over an organic kidneys, but is easy to produce."
```
Comment (Tayyyyyyy) [inline code/modules/surgery/organs/kidneys.dm]: I think this should say "It has no benefits over organic kidneys"

Comment (taukausanake) [reply code/modules/surgery/organs/kidneys.dm]: Do kidneys even DO anything? I can imagine maybe they remove some tox damage but I'd also imagine the liver to do that

**Comment by KasparoVy** [comment]
Would it also be reasonable to enable lungs, kidneys and livers for selection as assisted/mechanical organs in character setup as well like eyes & hearts currently are?

**Comment by Fox-McCloud** [comment]
We're kinda trying to move to that; right now there's 3 different mechanical organ types; ideally there would only be one, and it would be the same type as the kind you print, once it's all said and done.

That said, these need to go in first, then how mechanical organs are handled needs to be unified---then we can finally move in that direction (and thank goodness; that means far less confusion).

**Pull request merged**